src/HW_models/nrfx_glue_bsim.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Provides a minimalistic nrfx_glue to be used with Nordic nrfx and bsim.
+ */
+
+#ifndef NRFX_GLUE_BSIM_H__
+#define NRFX_GLUE_BSIM_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Macro for placing a runtime assertion.
+ *
+ * @param expression Expression to be evaluated.
+ */
+#ifndef NRFX_ASSERT
+#define NRFX_ASSERT(expression)
+#endif
+
+/* For BSIM we require redefining the pointers to the peripherals. */
+#include "nrfx_bsim_redef.h"
```

Comment:
The comment is unsuitable for `nrfx_bsim_redef.h`.

Reply:
Fixed